src/workloads/scale/MixedWorkloadsGenny.yml

```diff
@@ -0,0 +1,365 @@
+#   - Phase: 9..10
+#     Nop: true
+
+# - Name: Remove_128
+#   Type: CrudActor
+#   Database: *dbname
+#   ClientName: Remove
+#   Threads: 128
+#   Phases:
+#   - Phase: 0..9
+#     Nop: true
+#   - ExternalPhaseConfig:
+#       Path: *PhasePath
+#       Key: RemovePhase
+
+- Name: Insert_1
+  Type: CrudActor
+  Database: *dbname
+  ClientName: Insert
+  Threads: 1
+  Phases:
+  - Phase: 0..1
+    Nop: true
+  - ExternalPhaseConfig:
+      Path: *PhasePath
+      Key: InsertPhase
+  - Phase: 3..10
+    Nop: true
+
+- Name: Insert_16
+  Type: CrudActor
+  Database: *dbname
+  ClientName: Insert
+  Threads: 16
+  Phases:
+  - Phase: 0..3
+    Nop: true
+  - ExternalPhaseConfig:
+      Path: *PhasePath
+      Key: InsertPhase
+  - Phase: 5..10
+    Nop: true
+
+# - Name: Insert_32
+#   Type: CrudActor
+#   Database: *dbname
+#   ClientName: Insert
+#   Threads: 32
+#   Phases:
+#   - Phase: 0..5
+#     Nop: true
+#   - ExternalPhaseConfig:
+#       Path: *PhasePath
+#       Key: InsertPhase
+#   - Phase: 7..10
+#     Nop: true
+
+# - Name: Insert_64
+#   Type: CrudActor
+#   Database: *dbname
+#   ClientName: Insert
+#   Threads: 64
+#   Phases:
+#   - Phase: 0..7
+#     Nop: true
+#   - ExternalPhaseConfig:
+#       Path: *PhasePath
+#       Key: InsertPhase
+#   - Phase: 9..10
+#     Nop: true
+
+# - Name: Insert_128
+#   Type: CrudActor
+#   Database: *dbname
+#   ClientName: Insert
+#   Threads: 128
+#   Phases:
+#   - Phase: 0..9
+#     Nop: true
+#   - ExternalPhaseConfig:
+#       Path: *PhasePath
+#       Key: InsertPhase
+
+- Name: Find_1
+  Type: CrudActor
+  Database: *dbname
+  ClientName: Query
+  Threads: 16
+  Phases:
+  - Phase: 0..1
+    Nop: true
+  - ExternalPhaseConfig:
+      Path: *PhasePath
+      Key: FindPhase
+  - Phase: 3..10
+    Nop: true
+
+- Name: Find_16
+  Type: CrudActor
+  Database: *dbname
+  ClientName: Query
+  Threads: 16
+  Phases:
+  - Phase: 0..3
+    Nop: true
+  - ExternalPhaseConfig:
+      Path: *PhasePath
+      Key: FindPhase
+  - Phase: 5..10
+    Nop: true
+
+# - Name: Find_32
+#   Type: CrudActor
+#   Database: *dbname
+#   ClientName: Query
+#   Threads: 32
+#   Phases:
+#   - Phase: 0..5
+#     Nop: true
+#   - ExternalPhaseConfig:
+#       Path: *PhasePath
+#       Key: FindPhase
+#   - Phase: 7..10
+#     Nop: true
+
+# - Name: Find_64
+#   Type: CrudActor
+#   Database: *dbname
+#   ClientName: Query
+#   Threads: 64
+#   Phases:
+#   - Phase: 0..7
+#     Nop: true
+#   - ExternalPhaseConfig:
+#       Path: *PhasePath
+#       Key: FindPhase
+#   - Phase: 9..10
+#     Nop: true
+
+# - Name: Find_128
+#   Type: CrudActor
+#   Database: *dbname
+#   ClientName: Query
+#   Threads: 128
+#   Phases:
+#   - Phase: 0..9
+#     Nop: true
+#   - ExternalPhaseConfig:
+#       Path: *PhasePath
+#       Key: FindPhase
+
+
+AutoRun:
+  Requires:
+    mongodb_setup:
+    - replica
+    - single-replica
+    - standalone
+    - replica-noflowcontrol
+    - replica-1dayhistory-15gbwtcache
```

Comment:
One more comment :) Could you run this on the new maintenance build variants?

Reply:
`linux-3-node-replSet-maintenance-events`

Reply:
Yeah, I was thinking about that. I'll add.